README.md

```diff
@@ -49,98 +51,22 @@ You can optionally configure a `logger` on the connection to ease
 debugging:
 
 ```ruby
-require "logger"
+require 'logger'
 $fauna = Fauna::Connection.new(
   secret: server_key,
   logger: Logger.new(STDERR))
 ```
 
-### Client Contexts
-
-The easiest way to work with a connection is to open up a *client
-context*, and then manipulate resources within that context:
-
-```ruby
-Fauna::Client.context($fauna) do
-  user = Fauna::Resource.create('users', email: "taran@example.com")
-  user.data["name"] = "Taran"
-  user.data["profession"] = "Pigkeeper"
-  user.save
-  user.delete
-end
-```
-
-By working within a context, not only are you able to use a more
-convenient, object-oriented API, you also gain the advantage of
-in-process caching.
-
-Within a context block, requests for a resource that has already been
-loaded via a previous request will be returned from the cache and no
-query will be issued. This substantially lowers network overhead,
-since Fauna makes an effort to return related resources as part of
-every response.
-
-### Fauna::Resource
-
-All instances of fauna classes have built-in accessors for common
-fields:
-
-```ruby
-Fauna::Client.context($fauna) do
-  user = Fauna::Resource.create('users', constraints: {"username" => "taran77"})
-
-  # fields
-  user.ref       # => "users/123"
-  user.ts        # => 2013-01-30 13:02:46 -0800
-  user.deleted?  # => false
-  user.constraints # => {"username" => "taran77"}
-
-  # data and references
-  user.data       # => {}
-  user.references # => {}
-
-  # resource events timeline
-  user.events
-end
-```
-
-Fauna resources must be created and accessed by ref, i.e.
-
-```ruby
-pig = Fauna::Resource.create 'classes/pigs'
-pig.data['name'] = 'Henwen'
-pig.save
-pig.ref # => 'classes/pigs/42471470493859841'
-
-# and later...
-
-pig = Fauna::Resource.find 'classes/pigs/42471470493859841'
-# do something with this pig...
-````
-
-## Rails Usage
-
-Fauna provides a Rails helper that sets up a default context in
-controllers, based on credentials in `config/fauna.yml`:
-
-```yaml
-development:
-  email: taran@example.com
-  password: secret
-  secret: secret_key
-test:
-  email: taran@example.com
-  password: secret
-```
-
-(In `config/fauna.yml`, if an existing server key is specified, the
-email and password can be omitted. If a server key is not
-specified, a new one will be created each time the app is started.)
+### Client
 
-Then, in `config/initializers/fauna.rb`:
+Now that we have a connection, we need to create a client. The standard
+way to do this is by creating a client with the connection:
 
 ```ruby
-require "fauna/rails"
+client = Fauna::Client.new($fauna)
```

Comment:
Since client is an object now, standard use shouldn't require creating a connection and then a client. hide the connection class.